akka-actor-typed/src/main/resources/reference.conf

```diff
@@ -25,14 +25,17 @@ akka.library-extensions += "akka.actor.typed.internal.adapter.ActorSystemAdapter
 akka.actor {
   serializers {
     typed-misc = "akka.actor.typed.internal.MiscMessageSerializer"
+    service-key = "akka.actor.typed.internal.receptionist.ServiceKeySerializer"
   }
 
   serialization-identifiers {
     "akka.actor.typed.internal.MiscMessageSerializer" = 24
+    "akka.actor.typed.internal.receptionist.ServiceKeySerializer" = 26
```

Comment:
25 was used by `akka.cluster.sharding.typed.internal.ShardingSerializer`, so 26

Reply:
btw do we fail eagerly when we detect that?

Reply:
binding twice to same id in config i mean